src/library/other/Hasql/Pool/Config/Config.hs

```diff
@@ -0,0 +1,26 @@
+module Hasql.Pool.Config.Config where
+
+import Hasql.Pool.Observation (Observation)
+import Hasql.Pool.Prelude
+
+-- | Configufation for Hasql connection pool.
+data Config = Config
+  { size :: Int,
+    acquisitionTimeout :: DiffTime,
+    agingTimeout :: DiffTime,
+    idlenessTimeout :: DiffTime,
+    connectionStringProvider :: IO ByteString,
+    observationHandler :: Observation -> IO ()
+  }
+
+-- | Reasonable defaults, which can be built upon.
+standard :: Config
```

Comment:
nitpick: I'd call this `defaults` as it's a general convention that I'd expect from a library.

Reply:
Fixed. Just mentioning that this function is not exposed in the API any way.